frontend-react/src/App.tsx

```diff
@@ -24,15 +24,20 @@ import { permissionCheck, reportReceiver } from "./webreceiver-utils";
 import { Upload } from "./pages/Upload";
 import { useIdleTimer } from "react-idle-timer";
 import { CODES, ErrorPage } from "./pages/error/ErrorPage";
+import GlobalContextProvider from "./components/GlobalContextProvider";
 
 const OKTA_AUTH = new OktaAuth(oktaAuthConfig);
 
 const App = () => {
-
     const history = useHistory();
-    const customAuthHandler = () => {
+
+    const customAuthHandler = (): void => {
         history.push("/login");
     };
+
+    const handleIdle = (): void => {
+        if (OKTA_AUTH.authStateManager._authState.isAuthenticated) OKTA_AUTH.signOut();
```

Comment:
we'll want to clear the global context here just to be safe

Reply:
Good catch! Totally forgot we logout more than one place. :joy: